src/app/app.component.ts

```diff
@@ -1,10 +1,11 @@
 import { Component } from '@angular/core';
 import { RouterOutlet } from '@angular/router';
+import { TodosComponent } from './components/todos/todos.component';
 
 @Component({
   selector: 'app-root',
   standalone: true,
-  imports: [RouterOutlet],
+  imports: [RouterOutlet, TodosComponent],
   templateUrl: './app.component.html',
   styleUrl: './app.component.scss'
```

Comment:
Here don't understand where is that "OnPush", I see that you are using TodosComponent in app.route.ts, but not sure what is the difference/better

Reply:
I added `OnPush` setting in `angular.json`, but not sure if that is what you mean

Reply:
OnPush is a more strict change detection mechanism - it runs less 'refreshes' than default one, but you need to use it properly to avoid 'stale' data in template (when properties changed, but not refreshed in DOM).
With 'signals' it is very easy and this approach will be default and maybe the only one in the future - so it make sense to use it right away when writing new code.

To set it for the component - you need to set Component decorator parameter: https://angular.dev/best-practices/skipping-subtrees#using-onpush

The setting in angular.json affect code generation by Angular CLI utility when you use it to create new code, f.e.: `ng generate component`